docs/security.rst

```diff
@@ -116,6 +116,95 @@ This process enables HTTPS, but still defaults to HTTP so |sr| instances can sti
 - Do a rolling bounce of the cluster
 
 
+.. _schema_registry_basic_http_auth
+
+Configuring the REST API for Basic HTTP Authentication
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+|sr| can be configured to require users to authenticate using a username and password via the Basic HTTP authentication mechanism.
+
+.. note:: If you're using Basic authentication, we recommended that you
+          :ref:`configure |sr| to use HTTPS for secure communication <schema_registry_http_https>`,
+          because the Basic protocol passes credentials in plain text.
+
+Use the following settings to configure |sr| to require authentication:
+
+::
+
+    authentication.method=BASIC
+    authentication.roles=some-schema-registry-id
+    authentication.realm=SchemaRegistry-Props
+
+The ``authentication.roles`` config defines a comma separated list of user roles. To be authorized
+to access |sr| an authenticated user must belong to at least one of these roles.
+
+The ``authentication.realm`` config must match a section within ``jaas_config.file``, which
+defines how the server authenticates users and should be passed as a JVM option during server start:
+
+.. code:: bash
+
+    $ export SCHEMA_REGISTRY_OPTS=-Djava.security.auth.login.config=/path/to/the/jaas_config.file
```

Comment:
```suggestion
    export SCHEMA_REGISTRY_OPTS=-Djava.security.auth.login.config=/path/to/the/jaas_config.file
```

Reply:
Once we got the cool "copy" feature on the docs site, we changed our policy to not prepend `$` to example commands.

Reply:
> Once we got the cool "copy" feature on the docs site

What "copy" feature? Curious to learn :)

Reply:
@vcrfxia It's the little UI affordance on the right-hand side of code snippets: 
![docs screenshot](https://user-images.githubusercontent.com/12521043/55513148-de952c00-5619-11e9-98b8-e2956848cfc2.png)